.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: bash
+services: docker
+
+before_install:
+  - docker-compose up -d
```

Comment:
Building the required containers via `docker-compose` command.

Let all containers be executed as background process.